students/hyeonjin/users/views.py

```diff
@@ -1,11 +1,10 @@
-import json
-import re
-import bcrypt
+import json, re, bcrypt, jwt
 
 from django.http  import JsonResponse
 from django.views import View
 
 from users.models import User
+from django.conf import settings
 
```

Comment:
```suggestion

from django.http  import JsonResponse
from django.views import View
from django.conf  import settings

from users.models import User

```

현진님 모듈 import 순서 및 컨벤션 수정해주세요 🙂

Reply:
앗! 맘이 급했습니다! 바로 수정하겠습니다:)